src/App.js

```diff
@@ -7,8 +8,24 @@ import { BrowserRouter, Link, Switch, Route } from "react-router-dom";
 function App() {
   return (
     <BrowserRouter>
-      <h2>Welcome to App Component...</h2>
+      <Switch> 
+        <Route path="/" exact component={Menu}></Route>
+        <Route path="/loginpage" component={LoginPage}></Route>
+        <Route path="/page1" component={Page1}></Route>
+        <Route path="/page2" component={Page2}></Route>
+        <Route path="/page3" component={Page3}></Route>
```

Comment:
add exact param to all Route component

Reply:
https://github.com/RahafSalah/second-react-app/pull/6